entity-types/ext-ftconnector/golden_metrics.yml

```diff
@@ -0,0 +1,18 @@
+syncstatus:
+  title: Sync Status
+  unit: MS
```

Comment:
Is the unit right? The status is time ellapsed?

Reply:
yes for now